docs/03-TechnicalInfrastructure.md

```diff
@@ -90,31 +94,35 @@ Jiv images produced by the imaging insertion pipeline are organized per
 
 #### The `logs` directory
 
-The logs of the scripts are created under `data/logs` in `/data/project`.
+The logs of the scripts are created under `data/logs` in `/data/$PROJECT`.
     
 ```
-## Content of the /data/project/data/logs directory
+## Content of the /data/$PROJECT/data/logs directory
 .
-|__ TarLoad.log
-    |__ DTIPrep_pipeline
-    |   |__ DTI_QC`date`.log
-    |   |__ DTI_QC`date`.log
-    |__ DTIPrep_register
-    |   |__ DTIregister`date`.log
-    |   |__ DTIregister`date`.log
-    |__ registerProcessed
-        |__ registerProcessed`date`.log
-        |__ registerProcessed`date`.log
+|__ TarLoad-`hh-mm-xxxxxx`.log
+|__ DTIPrep_pipeline*
+|   |__ DTI_QC`date`.log
+|   |__ DTI_QC`date`.log
+|__ DTIPrep_register*
+|   |__ DTIregister`date`.log
+|   |__ DTIregister`date`.log
+|__ registerProcessed*
+    |__ registerProcessed`date`.log
+    |__ registerProcessed`date`.log
 ```
 
+`*` _denotes optional directories that are not automatically created by the 
```

Comment:
I think they are automatically created by the`DTIPrep_pipeline.pl, `DTIPrepRegister.pl` and  `register_processed_data.pl` scripts.

Reply:
based on the two scripts you mention, 
I can only see the script creating a `logs/DTIPrep_pipeline/` and `/logs/DTIPrep_register/` using `mkdir -p -m` which creates the directory, and another script is creating the `protocols/` directory as follows:
```my $tooldir = $data_dir . "/protocols/" . $tool;
    `mkdir -m 770 $tooldir`    unless (-e $tooldir)```
But it only has `-m` option, no `-p` option, so am not sure anymore...
I updated the documentation based on what I see in the scripts, but feel free to give me specific instrustioins